content/applications/general/email_communication/email_servers.rst

```diff
@@ -1,222 +1,209 @@
-   to make it work.
-
-How to manage outbound messages
-===============================
-As a system admin, go to :menuselection:`Settings --> General Settings`
-and check *External Email Servers*. Then, click *Outgoing Mail Servers* to create one and reference 
-the SMTP data of your email server. Once all the information has been filled out, click on 
-*Test Connection*.
+   If no one in the company is used to managing email servers, Odoo Online and Odoo.sh are strongly
+   recommended. Those Odoo hosting types' email systems work instantly and are monitored by
+   professionals. Nevertheless, a company can use their own email server if they want to manage the
+   email server's reputation themselves.
+
+The following sections contain information on how to integrate an external email server with Odoo.
+
+.. note::
+   Office 365 email servers don't easily allow the sending of external emails from hosts like Odoo.
+   Refer to `Microsoft's documentation <https://support.office.com/en-us/article/How-to-set-up-a-
+   multifunction-device-or-application-to-send-email-using-Office-365-69f58e99-c550-4274-ad18-
+   c805d654b4c4>`_ to make it work.
+
+Manage outbound messages
+========================
+
+As a system admin, go to :menuselection:`Settings --> General Settings --> Discuss` in Odoo, and
+enable the :guilabel:`External Email Servers` option. Then, click :guilabel:`Save`. Next, click
+:guilabel:`Outgoing Email Servers` and click :guilabel:`Create` to create a new outgoing mail
+server record in Odoo. Reference the SMTP data of the external email server. Once all the
+information has been filled out, click :guilabel:`Test Connection`.
 
 Here is a typical configuration for a G Suite server.
 
 .. image:: email_servers/outgoing-server.png
-    :align: center
+   :align: center
+   :alt: The typical G Suite configuration on Odoo.
 
-Then set your email domain name in the General Settings.
+Then, go back to :menuselection:`Settings --> General Settings --> Discuss` and set the email
+domain name in the :guilabel:`Alias Domain` name. Finally, click :guilabel:`Save` to finish setting
+up the email server.
 
 .. note::
-   If you get a ``[AUTHENTICATIONFAILED] Invalid credentials (Failure)`` warning when you *Test
-   Connection* on a Gmail address, activate the *Less secure app access* option. A direct link can
-   be `accessed here <https://myaccount.google.com/lesssecureapps?pli=1>`_.
+   If an `[AUTHENTICATION FAILED] Invalid credentials (Failure)` warning appears when
+   :guilabel:`Test Connection` is clicked for a Gmail address, activate the :guilabel:`Less secure
+   app access` option. A direct link can be `accessed here <https://myaccount.google.com/
+   lesssecureapps?pli=1>`_.
+
+   In addition to that, enable the :guilabel:`IMAP setting` on the Gmail account.
 
-   In addition to that, enable the IMAP setting on your Gmail account.
+Can an Office 365 server be used
+--------------------------------
```

Comment:
This heading is strange since it's a question but there's no question mark.
Suggestion below to match the format of the G Suite heading that immediately follows.
```suggestion
Use an Office 365 server
------------------------
```